components/grid/overview.md

```diff
@@ -201,7 +201,102 @@ You can define your own content for column cells or even the entire row through
 
 You can also set the [`Height` of the grid]({%slug common-features/dimensions%}), and you can use the `Class` to provide more complex CSS rules (like ones that will be inherited in a template).
 
```

Comment:
Mention the examples that are coming - conditional formatting and elastic (smaller) font/size